tests/10-issue.js

```diff
@@ -26,95 +26,332 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+            'representation of the GZIP-compressed [RFC1952] bitstring ' +
+            'values for the associated range of verifiable credential status ' +
+            'values.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20encodedList%20property%20of%20the%20credential%20subject%20MUST%20be%20a%20Multibase%2Dencoded%20base64url%20(with%20no%20padding)%20%5BRFC4648%5D%20representation%20of%20the%20GZIP%2Dcompressed%20%5BRFC1952%5D%20bitstring%20values%20for%20the%20associated%20range%20of%20verifiable%20credential%20status%20values.';
             const {slc: {credentialSubject}} = await getSlc({issuedVc});
             const {encodedList} = credentialSubject;
             // Uncompress encodedList
             const decoded = await sl.decodeList({encodedList});
-            should.exist(decoded);
-            // decoded size should be 16kb. For more info see
-            // `credentialSubject.encoded` in
-            // eslint-disable-next-line max-len
-            // https://w3c-ccg.github.io/vc-bitstring-status-list/#BitstringStatuslistCredential
-            const decodedSize = (decoded.length / 8) / 1024;
-            decodedSize.should.equal(16);
-          });
-      });
-    }
-  });
-
-describe('BitstringStatusList Credentials (Issue "statusPurpose: suspension")',
-  function() {
-    this.matrix = true;
-    this.report = true;
-    this.implemented = [...match.keys()];
-    this.rowLabel = 'Test Name';
-    this.columnLabel = 'Issuer';
-    for(const [issuerName, {issuers}] of match) {
-      describe(issuerName, function() {
-        let issuerResponse;
-        let err;
-        let issuedVc;
-        before(async function() {
-          const issuer = issuers.find(issuer => issuer.tags.has('Suspension'));
-          const {result, error, data} = await issueVc({issuer});
-          err = error;
-          issuerResponse = result;
-          issuedVc = data;
-        });
-        it('MUST issue a VC with BitstringStatusList "credentialStatus" and ' +
-          '"suspension" status purpose', async function() {
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          should.exist(issuerResponse);
-          should.not.exist(err);
-          issuerResponse.status.should.equal(201);
-          should.exist(issuedVc, `Expected VC from ${issuerName} to exist.`);
-          testCredential({credential: issuedVc});
-        });
-        // ensure that issued StatusList Credential contain correct properties
-        it('MUST have correct properties when dereferencing ' +
-          '"credentialStatus.statusListCredential"', async function() {
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          const {slc} = await getSlc({issuedVc});
-          testSlCredential({slCredential: slc});
-        });
-        it('MUST be 16kb in size when "encodedList" is decoded',
+            should.exist(decoded,
```

Comment:
`.exist` is a property, not a method, in Chai (afaik):
https://www.chaijs.com/api/bdd/#exist

Additionally, it seems there's a good bit more to test here with more specificity.

Reply:
I'm assuming that if the `sl.decodeList({encodedList})` returns a response, the encoding was done properly since that `decodeList` function will only work if encoded properly.

Reply:
I'm not sure what action you want me to take with the exist statement as it serves the intended purpose, to make sure the value exists